lunar/distribution.yaml

```diff
@@ -58,6 +58,26 @@ repositories:
       url: https://github.com/ros/angles.git
       version: master
     status: maintained
+  ar_track_alvar:
+    doc:
+      type: git
+      url: https://github.com/sniekum/ar_track_alvar.git
+      version: kinetic-devel
+    release:
+      packages:
+      - ar_track_alvar
+      - ar_track_alvar_metapkg
+      - ar_track_alvar_msgs
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/ros-gbp/ar_track_alvar-release.git
+      version: 0.7.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Sorry, it's probably not yet.
We're discussing to fork the repo to `ros-*` org unit https://github.com/sniekum/ar_track_alvar/issues/122. Is it too late if I take care of this after the necessary move is done (I'm expecting it to happen fairly soon)?